agent/src/util/discovery_operator.rs

```diff
@@ -218,6 +222,25 @@ impl DiscoveryOperator {
                 result = stream.get_message() => {
                     let response = result?.ok_or_else(|| anyhow::anyhow!("Received response type None. Should not happen."))?;
                     trace!("internal_do_discover - got discovery results {:?}", response.devices);
+
+                    //if device external information update is detected, simply remove the old akri instance
```

Comment:
Is the commented code needed? Also, should have space between forward slash and comment text and start with capitalized letter. `cargo fmt` may help with some of this but i am not sure.

Reply:
The comment is added for the newly added code. When external device information is updated in device inventory, the Akrii should be recreated to trigger new broker pod with latest external information. I will change the format.